src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -310,6 +310,7 @@ Apply to thead to make headers not bold.
 
 .text-preserve-space {
     white-space: pre-wrap;
+    word-break: break-all;
```

Comment:
I'm I right that `break-all` means the line wrapping can occur anywhere, even like this:

```
This is a line of te
xt.
```

If this is the case, we are just replacing the current problem with another one.